webmention.gemspec

```diff
@@ -1,34 +1,36 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
 require 'webmention/version'
 
-Gem::Specification.new do |s|
-  s.name          = 'webmention'
-  s.version       = Webmention::VERSION
-  s.date          = '2014-05-25'
-  s.homepage      = 'https://github.com/indieweb/mention-client-ruby'
-  s.summary       = 'A gem for sending webmention (and pingback) notifications'
-  s.authors       = [
-    'Aaron Parecki',
-    'Nat Welch'
-  ]
+Gem::Specification.new do |spec|
+  spec.required_ruby_version = ['>= 2.2.9', '< 2.6']
```

Comment:
Setting min and max supported versions. This is a breaking change from the existing setting: `'>= 1.9.3'`.

Ruby versions prior to 2.2.9 are no longer supported (see: https://www.ruby-lang.org/en/downloads/). 2.2.9 is in maintenance mode and will be end-of-life'd in the near future, but it's worth supporting for now.